packages/cli/src/commands/lockedgold/authorize.ts

```
@@ -21,29 +20,30 @@ export default class Delegate extends BaseCommand {
   static args = []
 
   static examples = [
-    'delegate --from=0x5409ED021D9299bf6814279A6A1411A7e866A631 --role Voting --to=0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d',
+    'authorize --from 0x5409ED021D9299bf6814279A6A1411A7e866A631 --role voter --to 0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d',
   ]
 
   async run() {
-    const res = this.parse(Delegate)
+    const res = this.parse(Authorize)
 
     if (!res.flags.role) {
       this.error(`Specify role with --role`)
       return
     }
 
     if (!res.flags.to) {
-      this.error(`Specify delegate address with --to`)
+      this.error(`Specify authorized address with --to`)
       return
     }
 
     this.kit.defaultAccount = res.flags.from
     const lockedGold = await this.kit.contracts.getLockedGold()
-    const tx = await lockedGold.delegateRoleTx(
-      res.flags.from,
-      res.flags.to,
-      Roles[res.flags.role as keyof typeof Roles]
-    )
-    await displaySendTx('delegateRoleTx', tx)
+    let tx: any
+    if (res.flags.role == 'voter') {
+      tx = await lockedGold.authorizeVoter(res.flags.from, res.flags.to)
+    } else if (res.flags.role == 'validator') {
+      tx = await lockedGold.authorizeValidator(res.flags.from, res.flags.to)
+    }
```

looks like invalid values for `role` fail silently?

---

I assumed `options: ['voter', 'validator']` would handle this for us, but I'll explicitly handle it here as well